console.rst

```diff
@@ -200,7 +200,7 @@ Now, you can pass the username to the command:
 Getting Services from the Service Container
 -------------------------------------------
 
-To actually create a new user, the command has to access to some
+To actually create a new user, the command has to have access to some
```

Comment:
In 2.8, the sentence reads like this: "[...] has to access some [...]" Shouldn't we changed it here accordingly?

Reply:
Oh, I see. Yes, I prefer 2.8's wording.